src/components/Page/LandingPageWidgets/MembersWidget.vue

```diff
@@ -44,6 +44,12 @@ export default {
 		NcButton,
 		SkeletonLoading,
 	},
+	props: {
```

Comment:
Is there a reason why you pass `currentCollective` as property instead of using the vuex getter like before?

Reply:
LandingPageWidgets now needs the currentCollective as well. I feel like passing it as a prop is the better pattern than accessing the vuex store twice. If disagreed I can change it back.

Reply:
Interesting. I always thought that accessing it from the store was cleaner because it meant less passing on of properties. I don't have strong opinion here though and am open to arguments why passing it as a property is superior :wink:

Reply:
changed Back, MembersWidget already has vuex getters anyways, so might as well just leave it like it is.